modules/tinymce/CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The nested menu item chevron icon was not fading when the menu item was disabled #TINY-7700
 - The table dialog did not always respect the `table_style_with_css` option #TINY-4926
 - Pasting into a table with multiple cells selected could cause the content to be pasted in the wrong location #TINY-7485
+- Mobile context menu was covering selection when more than one line was selected #TINY-7746
```

Comment:
This was only something changed within 5.9, so we shouldn't have a TinyMCE changelog entry for this. Also remember it's all selection anchors, and not just applicable to mobile.